renderdoc/driver/gl/gl_shader_refl.cpp

```diff
@@ -283,7 +283,7 @@ GLuint MakeSeparableShaderProgram(WrappedOpenGL &gl, GLenum type, vector<string>
   gl.glGetProgramiv(sepProg, eGL_LINK_STATUS, &status);
 
   // allow any vertex processing shader to redeclare gl_PerVertex
-  if(status == 0 && type != eGL_FRAGMENT_SHADER && type != eGL_COMPUTE_SHADER)
+  if(!IsGLES && status == 0 && type != eGL_FRAGMENT_SHADER && type != eGL_COMPUTE_SHADER)
```

Comment:
Amazing - I wish they had made that change to GL a long time ago. It's too late now, but it's such a pain to have to patch shaders :(.

Good that it's not required though, thanks.